pyfolio/timeseries.py

```diff
@@ -896,10 +896,10 @@ def get_max_drawdown_underwater(underwater):
 
     valley = np.argmin(underwater)  # end of the period
     # Find first 0
-    peak = underwater[:valley][underwater[:valley] == 0].index[-1]
+    peak = underwater[:valley].loc[underwater[:valley] == 0].index[-1]
```

Comment:
Do you mind elaborating how this fixes the `SystemError` that you got?

Reply:
let me find the root cause and explain it here.